lib/OpenQA/WebAPI/Controller/API/V1/Iso.pm

```diff
@@ -235,36 +235,33 @@ sub _generate_jobs {
 sub job_create_dependencies {
     my ($self, $job, $testsuite_mapping) = @_;
 
+    my @error_messages;
     my $settings = $job->settings_hash;
-    for my $depname ('START_AFTER_TEST', 'PARALLEL_WITH') {
+    for my $dependency (
+        {'START_AFTER_TEST', OpenQA::Schema::Result::JobDependencies::CHAINED},
```

Comment:
I mean to write a tuple, but there are no tuples in Perl. So square brackets to emulate tuple with an array?